releasenotes/sfdx/README.md

```diff
@@ -21,10 +21,99 @@ Additional documentation:
+        {
+          "filename": "force-app/main/default/classes/myClass.cls",
+          "stringToReplace": "replaceMe",
+          "replaceWithEnv": "THE_REPLACEMENT"  
+        }
+      ],
+    …
+    }
+    ```
+
+    You can specify these keys in the `replacements` property:
+
+    * `filename`: Single file that contains the string to be replaced.
+    * `glob`: Collection of files that contain the string to be replaced. Example: `**/classes/*.cls`.
+    * `stringToReplace`: The string to be replaced.
+    * `regexToReplace`: Regular expression that specifies a string pattern to be replaced. 
+    * `replaceWithEnv`: Specifies that the string be replaced with the value of the environment variable.
+    * `replaceWithFile`: Specifies that the string be replaced with the contents of a file.
+    * `replaceWhenEnv`: Specifies a condition, using environment variables, for when a string replacement occurs. 
+
+    This example is similar to the previous one, except that the replacement occurs only if an environment variable called `DEPLOY_DESTINATION` exists and it has a value of `PROD`.
+
+    ```bash 
+    "replacements": [
+      {
+        "filename": "force-app/main/default/classes/myClass.cls",
+        "stringToReplace": "replaceMe",
+        "replaceWithEnv": "THE_REPLACEMENT"
+        "replaceWhenEnv": [{
+          "env": "DEPLOY_DESTINATION",
+          "value": "PROD"
+        }]  
+      }
+    ],
+    ```
+
+    We’re updating the [Salesforce DX Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_develop.htm) with more details and examples; we’ll let you know when it’s available.
+
+* NEW: Retrieve source files into a non-package directory (AKA a directory that _isn't_ configured in your `sfdx-project.json` file) with the new `--retrievetargetdir` parameter of the `force:source:retrieve` command. With this parameter you can now keep your unpackaged source files separate from your packaged files. Then, for example, you can easily prevent these unpackaged files from being deployed to a scratch org because they're not included in any configured package directory. 
+
+    This example shows how to retrieve all Apex classes from your default org and put the source-formatted files into the `unpackaged-files` directory. If this directory doesn't exist, the command creates it for you. Your configured package directories are unchanged.
+
+    `sfdx force:source:retrieve --retrievetargetdir ./unpackaged-files --metadata ApexClass`
+
+    Many thanks to [Matthias Rolke](https://github.com/amtrack) for suggesting the cool feature, and then writing a lot of the code!  [plugin-source PR #426](https://github.com/salesforcecli/plugin-source/pull/426)
+
+* CHANGE: We upgraded the version of Node.js contained in the full Salesforce CLI Docker image to LTS v18. (sfdx-cli PR [#720](https://github.com/salesforcecli/sfdx-cli/pull/720))
+
+* FIX: The `force:source:*` commands now support these metdata types:
```

Comment:
they tried to add these previously, and had the names slightly wrong.  I can't remember if those went out last release or not.  If not, they're net new and all good.

Reply:
yeah, I noticed that. And yeah, I doc'd 'em last release.  I'll figure out which names changed and list them that way. Thx.